HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -177,6 +177,30 @@ def customiseFor2017DtUnpacking(process):
 
     return process
 
+def customiseForMuonGeometry(process):
```

Comment:
Please add the PR number here in the name of the customisation fucntion - customiseFor30280